lib/pacemaker/pcmk_sched_instances.c

```diff
@@ -695,40 +695,49 @@ find_compatible_instance_on_node(const pe_resource_t *match_rsc,
     return NULL;
 }
 
+/*!
+ * \internal
+ * \brief Find a clone instance or bundle container compatible with a resource
+ *
+ * \param[in] match_rsc  Resource that instance must match
+ * \param[in] rsc        Clone or bundle resource to check for matching instance
+ * \param[in] role       If not RSC_ROLE_UNKNOWN, instance must match this role
+ * \param[in] current    If true, compare instance's original node and role,
+ *                       otherwise compare assigned next node and role
+ *
+ * \return \p rsc instance matching \p node and \p role if any, otherwise NULL
```

Comment:
We don't have a `\p node` in this one; we're grabbing `match_rsc`'s node (or allowed nodes) and matching against that in the called function

Reply:
thanks